### PR TITLE
VDS-638: Migrate product page

### PR DESCRIPTION
**Pull request by ilyawzrd**
Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-demo-theme-b2b-1.11.0-pr-177.zip

### CODE REVIEW

File: snippets/bootstrap-migration/product/type/configured/quantity.liquid [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        ng-model="configurationQty"
+        ng-change="quantityChanged(configurationQty)"
+        ng-model-options="{ debounce: 300 }"
+        >
+        <span class="input-group-append">
+            <button class="btn btn-outline-primary border-1 rounded-0"
+                    {% if unavailable or sold_out %} disabled {% endif %}
+                    title="{{ 'products.general.add_to_cart' | t }}"
+                    ng-click="addSelectedProductsToCart()">
+                        {% if unavailable %}
```
Comment (asvishnyakov) [inline snippets/bootstrap-migration/product/type/configured/quantity.liquid:16]: Please fix formatting

File: snippets/bootstrap-migration/product/price.liquid [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    <dd class="font-weight-bold">
+        <span class="{% if variation_price %}{% else %}text-dark-green{% endif %}">
+            {% if variant != null %}
+                {% assign productOrVariant = variant %}
+            {% else %}
+                {% assign productOrVariant = product %}
+            {% endif %}
+            {% if settings.show_prices_with_taxes %}
+                {% assign price = productOrVariant.price_with_tax %}
```
Comment (asvishnyakov) [inline snippets/bootstrap-migration/product/price.liquid:21]: Fix formating please

File: snippets/bootstrap-migration/dialogs/recently-added-cart-item.liquid [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                            type="number" class="form-control text-center"
+                            min="1" max="{% raw %}{{ maxConfigurationQty }}{% endraw %}"
+                            ng-model="configurationQty"
+                            ng-change="quantityChanged(configurationQty)"
+                            required
+                            >
+                            <span ng-if="dialogData.inventoryError" class="text-muted small">{% raw %}{{ item.availableQuantity }}{% endraw %}&nbsp;{{ 'cart.general.in_stock' | t }}</span>
+                        </td>
+                        <td class="col-2 text-right border-0">
+                            {% if settings.show_prices_with_taxes %}
```
Comment (asvishnyakov) [inline snippets/bootstrap-migration/dialogs/recently-added-cart-item.liquid:48]: Fix formatting

File: assets/js/product-compare/add-to-compare-checkbox.js [hunk 4]
```diff
@@ -1,51 +1,66 @@
...
+  $ctrl.addProductToCompareList = function (event) {
+      event.preventDefault();
+      catalogService.getProduct($ctrl.productId).then(function(response) {
+          var product = response.data[0];
+          event.preventDefault();
+          var isInProductList = compareProductService.isInProductCompareList($ctrl.productId);
+          if (!isInProductList) {
+              var count = compareProductService.getProductsCount();
+              if (count < 5) {
+              $ctrl.containProduct = true;
```
Comment (asvishnyakov) [inline assets/js/product-compare/add-to-compare-checkbox.js:19]: Fix formatting

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: assets/js/product-compare/add-to-compare-checkbox.js (hunk 4 above)
Comment (ilyawzrd) [reply assets/js/product-compare/add-to-compare-checkbox.js]: done

File: snippets/bootstrap-migration/product/type/configured/quantity.liquid (hunk 1 above)
Comment (ilyawzrd) [reply snippets/bootstrap-migration/product/type/configured/quantity.liquid]: done

File: snippets/bootstrap-migration/product/price.liquid (hunk 2 above)
Comment (ilyawzrd) [reply snippets/bootstrap-migration/product/price.liquid:21]: I there's nothing to fix in this file

File: snippets/bootstrap-migration/dialogs/recently-added-cart-item.liquid (hunk 3 above)
Comment (ilyawzrd) [reply snippets/bootstrap-migration/dialogs/recently-added-cart-item.liquid:47]: done

File: snippets/bootstrap-migration/product/price.liquid (hunk 2 above)
Comment (asvishnyakov) [reply snippets/bootstrap-migration/product/price.liquid:21]: Agree, probably I was confused by `assign` and `include`

**Review by asvishnyakov — APPROVED** [review]

**Review by trueboroda — APPROVED** [review]

**Pull request merged**